packages/onboarding-ui/src/pages/RequestTrialPage/RequestTrialPage.tsx

```diff
@@ -26,6 +25,9 @@ const RequestTrialPage = (props: RequestTrialPageProps): ReactElement => {
         subtitle={t('page.requestTrial.subtitle')}
       >
         <RequestTrialForm {...props} />
+        <ManageWorkspaceFallback
+          onManageWorkspaceClick={props.onManageWorkspaceClick}
+        />
```

Comment:
This was not supposed to come back.  This was intentionally removed: https://github.com/RocketChat/Rocket.Chat.Fuselage/pull/566/files